Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -164,7 +164,7 @@ WarpX::AddSpaceChargeField (WarpXParticleContainer& pc)
     computePhi( rho, phi, beta, pc.self_fields_required_precision, pc.self_fields_max_iters, pc.self_fields_verbosity );
 
     // Compute the corresponding electric and magnetic field, from the potential phi
-    computeE( Efield_fp, phi, beta );
+    // computeE( Efield_fp, phi, beta );
```

Comment:
This doesn't look correct? I guess we just need @roelof-groenewald 's electrostatic refactor / gradient calculation, but let's make sure this doesn't introduce an additional change or reversion.

Reply:
Thanks for pointing this out, I'm not sure how this happened.